diff --git a/package/periodic-11.1/scripts/daily/100.clean-disks b/package/periodic-11.1/scripts/daily/100.clean-disks
index b4ebf30..310354c 100755
--- a/package/periodic-11.1/scripts/daily/100.clean-disks
+++ b/package/periodic-11.1/scripts/daily/100.clean-disks
@@ -7,9 +7,9 @@
 
 # If there is a global system configuration file, suck it in.
 #
-if [ -r /etc/defaults/periodic.conf ]
+if [ -r /etc/default/periodic.conf ]
 then
-    . /etc/defaults/periodic.conf
+    . /etc/default/periodic.conf
     source_periodic_confs
 fi
 
@@ -41,7 +41,8 @@ case "$daily_clean_disks_enable" in
 		    print=;;
 	    esac
 
-	    rc=$(find / \( ! -fstype local -o -fstype rdonly \) -prune -o \
+	    # search all files excluding mounted locations and readonly files
+	    rc=$(find / -mount \( ! -perm /a+w \) -prune -o \
 		\( $args \) -atime +$daily_clean_disks_days \
 		-execdir rm -df {} \; $print | tee /dev/stderr | wc -l)
 	    [ -z "$print" ] && rc=0
